lib/just_audio_media_kit.dart

```diff
@@ -40,15 +40,22 @@ class JustAudioMediaKit extends JustAudioPlatform {
   /// This uses `scaletempo` under the hood & disables `audio-pitch-correction`.
   static bool pitch = true;
 
-  /// Enables gapless playback via the [`--prefetch-playlist`](https://mpv.io/manual/stable/#options-prefetch-playlist) in libmpv
+  /// Enables gapless playback via the [`--prefetch-playlist`](https://mpv.io/manual/stable/#options-prefetch-playlist) in libmpv.
   ///
   /// This is highly experimental. Use at your own risk.
   ///
   /// Check [mpv's docs](https://mpv.io/manual/stable/#options-prefetch-playlist) and
-  /// [the related issue](https://github.com/Pato05/just_audio_media_kit/issues/11) for more information
+  /// [the related issue](https://github.com/Pato05/just_audio_media_kit/issues/11) for more information.
+  ///
+  /// [prefetchPlaylistSize] can be changed to set the amount of items to prefetch.
   static bool prefetchPlaylist = false;
 
-  static final _logger = Logger('JustAudioMediaKit');
+  /// Max amount of items to prefetch in the playlist.
+  ///
+  /// Does nothing, if [prefetchPlaylist] is set to false. Default is 2.
+  /// It is not recommended to change this, because libmpv doesn't prefetch more than 2 items.
```

Comment:
> because libmpv doesn't prefetch more than 2 items.

Did you test this? As far as my understanding goes, I remember libmpv fetches data till the cache limit is hit, which should be of 150MB (according to my older comment https://github.com/Pato05/just_audio_media_kit/issues/11#issuecomment-2045077101, though I do not remember where I got that information from).

Reply:
I still do not remember where I got the 150MB from, but it seems like the default cache size might actually be 32MB (default `bufferSize`), since `bufferSize` sets the properties [`--demuxer-max-bytes`](https://mpv.io/manual/master/#options-demuxer-max-bytes) and [`--demuxer-max-back-bytes`](https://mpv.io/manual/master/#options-demuxer-max-back-bytes), which seem to be tied to [`--prefetch-playlist`](https://mpv.io/manual/master/#options-prefetch-playlist) as stated in docs: "_and depends on the demuxer cache settings_"

Reply:
I tested myself, and yes, it seems like libmpv fetches only the next item in playlist (tested this without the shuffle implementation).